examples/step-74/step-74.cc

```diff
@@ -351,12 +351,12 @@ namespace Step74
           {
             for (unsigned int j = 0; j < fe_v.dofs_per_cell; ++j)
               copy_data.cell_matrix(i, j) +=
-                // \nu \nabla u \nabla v
-                diffusion_coefficient * fe_v.shape_grad(i, point) *
-                fe_v.shape_grad(j, point) * JxW[point];
+                diffusion_coefficient *
+                fe_v.shape_grad(i, point) *             // nu grad v_h
+                fe_v.shape_grad(j, point) * JxW[point]; // grad u_h dx
 
-            copy_data.cell_rhs(i) +=
-              rhs[point] * fe_v.shape_value(i, point) * JxW[point];
+            copy_data.cell_rhs(i) += rhs[point] * fe_v.shape_value(i, point) *
+                                     JxW[point]; // f * v_h * dx
```

Comment:
Can you try playing with the formatting a little bit to make these lines look a little nicer? I would put the dx on a separate line, for example.

Reply:
I reformatted them, do you think they are better now ?

Reply:
Also, while you're there, put the phi_i first -- we like to multiply with the test function from the left :-)

Reply:
Good point, I have moved all the test functions to the left.